README.md

```diff
@@ -7,7 +7,12 @@
 ![Docs](https://github.com/luxonis/luxonis-ml/actions/workflows/docs.yaml/badge.svg)
 ![Coverage](media/coverage_badge.svg)
 
-A collection of helper function and utilities.
+This library includes a collection of helper functions and utilities for the Luxonis MLOps stack. This includes the following submodules:
+
+- [Dataset Management](./luxonis_ml/data): Creating computer vision datasets focused around Luxonis hardware and loading data into our training library.
```

Comment:
Do the links make sense since there are no READMEs anymore?

Reply:
They are meant to link to the source code instead of just a README but I will remove them since I don't think its necessary and maybe confusing.